include/kernel/kernel.hpp

```diff
@@ -104,6 +104,8 @@ class Kernel {
 	std::string getProcessName(u32 pid);
 	const char* resetTypeToString(u32 type);
 
+	int copyStringFromUser(u8 *dst, u32 src, u32 size);
```

Comment:
Rather than `int` maybe it's time for `i32`/`i16`/`i8`-types.

Reply:
Yeah s32 sounds good